specs/phase0/deposit-contract.md

```diff
@@ -31,12 +31,16 @@ This document represents the specification for the beacon chain deposit contract
 
 | Name | Value |
 | - | - |
+| `DEPOSIT_CHAIN_ID` | `1` |
+| `DEPOSIT_NETWORK_ID` | `1` |
```

Comment:
Very nitpicking:
This table is called "Contract". Maybe move `DEPOSIT_CHAIN_ID` and `DEPOSIT_CHAIN_ID` to a "Chain" table or remove the markdown header "Contract".

Reply:
made a "constants" and "config" section. should be good now